### PR TITLE
Cybersource cron job for folio

### PR DESCRIPTION
**Pull request by jgreben**
This changes a lot of what was done before, but simplifies a few things. It may be useful to pair in order to review.

### CODE REVIEW

**Comment by jgreben** [comment]
@shelleydoljack I think this is ready for review as a real PR.

File: Gemfile.lock [hunk 1]
```diff
@@ -138,40 +115,45 @@ GEM
...
 
 DEPENDENCIES
   byebug
-  capistrano
-  capistrano-bundler
-  capistrano-shared_configs
-  config
   cybersource_rest_client (= 0.0.31)
-  dlss-capistrano
+  folio_client (~> 0.13.0)
```
Comment (shelleydoljack) [inline Gemfile.lock:146]: nice

File: cyber_source/download_payment_batch_detail_report.rb [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        puts e.message
+        next
+      end
+      next unless data
+
+      data.each_line.with_index do |batch_detail, index|
+        if index > 1
+          user_id = batch_detail.split(',')[4]
+          paid = batch_detail.split(',')[8]
+          paydate = batch_detail.split(',')[12]&.chomp
```
Comment (shelleydoljack) [inline cyber_source/download_payment_batch_detail_report.rb:56]: Is paydate in UTC? I think we need to account for folio storing the date in UTC when reconciling folio account createdDate with the batch_detail paydate.

**Review by shelleydoljack — COMMENTED** [review]
Seems okay except I wonder about the date comparison, especially since folio stores it as UTC. Won't we miss things that were paid after 4pm PST? Also, I don't understand what the csv is for. Is it going to be to update FOLIO or for humans to look at? If it's for humans, the UUID's should be resolved.

File: cyber_source/download_payment_batch_detail_report.rb (hunk 2 above)
Comment (jgreben) [reply cyber_source/download_payment_batch_detail_report.rb:56]: Cybersource reports the payment time in UTC (Note the 'Z' in the timestamp in the spec/fixtures file)

**Comment by jgreben** [comment]
> Seems okay except I wonder about the date comparison, especially since folio stores it as UTC. Won't we miss things that were paid after 4pm PST? Also, I don't understand what the csv is for. Is it going to be to update FOLIO or for humans to look at? If it's for humans, the UUID's should be resolved.

The CSV is to be mailed to the finance folks so they can do the journal entries. They may need to create a pivot table in order to view the data grouped by payment reason, and they already know about this. They do not need to use the UUID, it is there as a reference to the account and if there is ever a question about a payment we can use it to look it up.

**Review by shelleydoljack — APPROVED** [review]

**Pull request merged**